### PR TITLE
Deprecate login command PS-9864

### PR DESCRIPTION
**Pull request by BartoszCki**

### CODE REVIEW

File: paperspace/cli/auth.py [hunk 1]
```diff
@@ -1,36 +1,35 @@
 import click
 
-from paperspace import client, config
+from paperspace import logger
 from paperspace.cli import common
 from paperspace.cli.cli import cli
-from paperspace.cli.validators import validate_email
 from paperspace.commands import login as login_commands
 
+LOGIN_DEPRECATION_MESSAGE = """The login command is currently disabled for logging in with `--email` and `--password`.
```
Comment (jaredscheib) [inline paperspace/cli/auth.py]:
```suggestion
LOGIN_DEPRECATION_MESSAGE = """The login command is currently disabled for logging in using `--email` and `--password`.
```

File: paperspace/cli/auth.py [hunk 2]
```diff
@@ -40,7 +41,7 @@ def logout():
 
 
 @cli.command("apiKey", help="Save your api key")
-@click.argument("api_key", required=False, callback=common.prompt_for_secret("API Key: "))
+@click.argument("api_key", required=False, callback=common.prompt_for_secret("Enter your API Key: "))
```
Comment (jaredscheib) [inline paperspace/cli/auth.py]: will this succeed if no string is provided?

File: paperspace/login.py [hunk 3]
```diff
@@ -125,6 +125,8 @@ def set_apikey(apikey):
         json.dump(config_data, outfile, indent=2, sort_keys=True)
         outfile.write('\n')
 
+    logger.log("Successfully added API Key to {}. You're ready to go!".format(config_path))
```
Comment (jaredscheib) [inline paperspace/login.py]:
```suggestion
    logger.log("Successfully added your API Key to {}. You're ready to go!".format(config_path))
```

File: paperspace/cli/auth.py (hunk 1 above)
Comment (BartoszCki) [reply paperspace/cli/auth.py]: done

File: paperspace/cli/auth.py (hunk 2 above)
Comment (BartoszCki) [reply paperspace/cli/auth.py]: done

File: paperspace/cli/auth.py (hunk 1 above)
Comment (jaredscheib) [reply paperspace/cli/auth.py]: 👍

File: paperspace/cli/auth.py (hunk 2 above)
Comment (jaredscheib) [reply paperspace/cli/auth.py]: 👍

Comment (jaredscheib) [reply paperspace/cli/auth.py]: ah yes, i meant actually, could we have it fail with `API Key cannot be empty.` if the trimmed input is empty? that way we offer some minimal amount of protection/validation

Comment (BartoszCki) [reply paperspace/cli/auth.py]: fixed

**Pull request merged**